sharpy/generators/gustvelocityfield.py

```diff
@@ -250,15 +249,17 @@ def initialise(self, in_dict):
         settings.to_custom_types(self.settings, self.settings_types, self.settings_default)
 
         self.file_info = np.loadtxt(self.settings['file'])
-
+        self.initialise_inteprolation_functions()
```

Comment:
*interpolation*

Reply:
Thanks for spotting! Fixed in https://github.com/ImperialCollegeLondon/sharpy/pull/217/commits/c75c9b4cbfad39f523c20a35016fcae24bd91c33